packages/devextreme/testing/testcafe/tests/editors/slider/slider.ts

```diff
@@ -9,7 +9,9 @@ fixture.disablePageReloads`Slider`
 test('Slider appearance', async (t) => {
   const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
 
-  await testScreenshot(t, takeScreenshot, 'slider-appearance.png', { element: '#container' });
+  await testScreenshot(t, takeScreenshot, 'slider-appearance.png', { element: '#container', theme: 'light' });
+
+  await testScreenshot(t, takeScreenshot, 'slider-appearance.png', { element: '#container', theme: 'dark' });
```

Comment:
We actually don't have `dark` have, we have `dx.material.blue.dark`, `dx.fluent.blue.dark` and so on. Please, get the theme name the way u did before last commits